api/timeline/v1/timeseries.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package clutch.timeline.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/timeline/v1;timelinev1";
+
+import "google/protobuf/any.proto";
+import "validate/validate.proto";
+
+message TimeseriesData {
```

Comment:
Can you write a comment about what this represents?  It looks like just a single datapoint but some more color here would be helpful to the reader.